app/templates/gold.html

```diff
@@ -5,27 +5,28 @@
 {% endblock headers %}
 
 {% block body %}
-<style>
-    body {
-    background-color: rgba(8, 247, 235, 0.842)
-    }
-</style>
-    <div class="col-md-9 px-0">
+    <div class="col-md-12 px-0">
     <h1 class="display-4" style="margin-top: 10px;">Gold Analysis</h1>
-    <p class="lead">
-        <h2 align="right">Gold Performance 7 Days after Incident</h2>
+    <p>The occurrence of an earthquake has minimal impact of the direction of Gold prices in the
+            subsequent 7-day, 14-day and 1-month intervals as indicated in the charts below. The actual
+            performances ranged between a drop of .087% and a gain of 1.73% after seven days; a drop of 1.48%
+            and a gain of 3.11% after 14 days and a drop of 1.49% and a gain of 2.95% after a month.
+            However in the vast majority of instances (Magnitude 7.1 or less), the performance averages
+            between flat and a gain of .01%.</p>
+   <p class="lead">
+        <h2 align="center">Gold Performance 7 Days after Incident</h2>
         <iframe width="1200" height="600" frameborder="0" scrolling="no" src="//plot.ly/~wel51x/144.embed"></iframe>
```

Comment:
```suggestion
        <iframe width="1200" height="600" frameborder="0" scrolling="no" src="//plot.ly/~wel51x/144.embed" style="margin-bottom: 10px"</iframe>
```

If you need extra space below an element, use the appropriate styling rather than empty `<p>` tags.